backend/leanstore/storage/buffer-manager/BufferFrame.hpp

```diff
@@ -43,25 +45,139 @@ struct BufferFrame {
       ContentionTracker contention_tracker;
       // -------------------------------------------------------------------------------------
       struct OptimisticParentPointer {
```

Comment:
Are you now using OptimisticParentPointer? It was used only in micro experiments

Reply:
I just added a guard to remove some race conditions. it still is behind a flag